eng/helix/content/runtests.cmd

```diff
@@ -24,10 +26,16 @@ set DOTNET_CLI_HOME=%HELIX_CORRELATION_PAYLOAD%\home
 
 set PATH=%DOTNET_ROOT%;!PATH!;%HELIX_CORRELATION_PAYLOAD%\node\bin
 echo Set path to: %PATH%
-echo "Invoking InstallDotNet.ps1 %$arch% %$sdkVersion% %$runtimeVersion% %DOTNET_ROOT%"
-powershell.exe -NoProfile -ExecutionPolicy unrestricted -file InstallDotNet.ps1 %$arch% %$sdkVersion% %$runtimeVersion% %DOTNET_ROOT%
+
+powershell.exe -noLogo -NoProfile -ExecutionPolicy unrestricted -command ". eng\common\tools.ps1; InstallDotNet %DOTNET_ROOT% %$sdkVersion% %$arch% '' $true"
+IF [%$feedCred%] == [] (
+    powershell.exe -noLogo -NoProfile -ExecutionPolicy unrestricted -command ". eng\common\tools.ps1; InstallDotNet %DOTNET_ROOT% %$runtimeVersion% %$arch% dotnet $true"
+) else (
+    powershell.exe -noLogo -NoProfile -ExecutionPolicy unrestricted -command ". eng\common\tools.ps1; InstallDotNet %DOTNET_ROOT% %$runtimeVersion% %$arch% dotnet $true https://dotnetclimsrc.blob.core.windows.net/dotnet %$feedCred%"
```

Comment:
Shoot. We need a new Arcade SDK containing some option to include `-noPath` in the `install-dotnet.ps1 command line. Could hack that into eng/common/tools.ps1 but that's a temporary bandaid at best. @MattGal is such an option planned for a near-term tools.ps1❔ If not, would Arcade accept a change supporting one of `$installParameters` or `$noPath` parameters to `InstallDotNet`❔

What's happening here is a clash between a change I made to handle private runtime locations and a change @HaoK made to avoid corrupting `$env:Path` on the Helix agents. We need both.

Reply:
I think someone commented about this in the first responders thread, so I think they are planning on adding that support (soonish?)

Reply:
/cc @JunTaoLuo

Reply:
Seems like we should just hack our copy of tools.ps1 for now?

Reply:
Unless we know the next Arcade dependency PR will update tools.ps1 to include a fix, that's scary @HaoK i.e. the hack will just get auto-undone.

Could you point me to the first responders thread please❔ I'm not seeing it.

Reply:
Last two comments of https://teams.microsoft.com/l/message/19:afba3d1545dd45d7b79f34c1821f6055@thread.skype/1597443105767?tenantId=72f988bf-86f1-41af-91ab-2d7cd011db47&groupId=4d73664c-9f2f-450d-82a5-c2f02756606d&parentMessageId=1597443105767&teamName=.NET%20Core%20Eng%20Services%20Partners&channelName=First%20Responders&createdTime=1597443105767

Reply:
That was about the diagnostics repo, not Arcade, at least as far as I can tell.

Reply:
oops, I had a typo in my earlier comment and didn't actually loop in @MattGal. I know @MattGal's working on tools.ps1 changes in a related area but haven't seen an issue about `InstallDotNet` supporting `-noPath`.

Reply:
Filed dotnet/arcade#6060 because I couldn't find an issue about this requirement.

Reply:
The CI is green - should we merge this now, and file a followup issue to fix the `-noPath` issue for later in rc1 or rc2? That way we can ensure linear git history is preserved from preview8 to rc1 without needing to rush to fix the `nopath` issue

Reply:
No, this isn't good to go because it'll regress dotnet/aspnetcore#23575. Wait for @MattGal to fix dotnet/arcade#6060.

Reply:
Now need another `$true`  at the end of this line and `'' '' $true` in the two commands above. As-is, we're not taking advantage of the new feature you copied from @MattGal's dotnet/arcade#6092.